dashboards-observability/public/components/event_analytics/explorer/visualizations/index.tsx

```diff
@@ -47,13 +48,13 @@ export const ExplorerVisualizations = ({
   handleOverrideTimestamp,
   callback,
   changeIsValidConfigOptionState,
+  queryManager,
 }: IExplorerVisualizationsProps) => {
   const { tabId } = useContext<any>(TabContext);
   const { data, vis } = visualizations;
   const { data: vizData = {}, metadata: { fields = [] } = {} } = data?.rawVizData;
-
   const fieldOptionList = explorerFields.availableFields.map((field) => {
-  // const fieldOptionList = fields.map((field) => {
+    // const fieldOptionList = fields.map((field) => {
```

Comment:
is commented code required in future? if yes then add as` TODO:` else please remove.